codechecker_lib/arg_handler.py

```diff
@@ -39,27 +39,21 @@ def handle_list_checkers(args):
     List the default enabled and disabled checkers in the config.
     """
     context = generic_package_context.get_context()
-    enabled_analyzers = args.analyzers
+    # If nothing is set, list checkers for all supported analyzers.
+    enabled_analyzers = args.analyzers or analyzer_types.supported_analyzers
```

Comment:
We do not need this type to be a list type. We only use this in iterations, so having it as iterable is good.